pandas/core/groupby/grouper.py

```diff
@@ -54,14 +54,16 @@ class Grouper(object):
     axis : number/name of the axis, defaults to 0
     sort : boolean, default to False
         whether to sort the resulting labels
-
-    additional kwargs to control time-like groupers (when `freq` is passed)
-
     closed : closed end of interval; 'left' or 'right'
+        Only when `freq` parameter is passed.
```

Comment:
Sorry just one last pass at all of these where `Only when freq...` is the description - not sure that's the best way of going about this.

It looks like the description for these is where the type should go. Can you move the description down to the line below and add a type for this and the 3 other parameters sharing this description? Also would be nice to revert removal of previous line if it doesn't cause PR04 failures

Reply:
What do you mean by sharing the description?

Reply:
I'll fix it in few hours. Sorry, busy weekend.

Reply:
I'll fix it in few hours. Sorry, busy weekend.

Reply:
I have no idea how deal with line below (for all kwargs).

> _Additional kwargs to control time-like groupers (when `freq` is passed)._

There is no description sharing functionality for Python docstrings.